nvflare/fuel/f3/drivers/connector.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2021-2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from dataclasses import dataclass
+from enum import Enum
+
+from nvflare.fuel.f3.drivers.net_utils import short_url
+
+
+class Mode(Enum):
+    ACTIVE = 0
+    PASSIVE = 1
+
+
+@dataclass
+class Connector:
```

Comment:
should we move this into upper layer?
``nvflare/fuel/f3``

Reply:
can we change the class name to be something like `ConnectorInfo` since it is not really a Connector but this class is about the attributes/information

Reply:
Good catch. Anything exposed to upper layer should be in f3.

Reply:
Got it, what about `DriverInfo`?
Is this class `ConnectorInfo` or `DriverInfo`?

Reply:
ConnectorInfo is better. ConnectorInfo is container for host and port etc. Driver is generic. When driver starts with connect/listen, ConnectorInfo is the argument.